setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    name="linked_list",
+    description="Week 2 Day 1 Assignment - Linked List",
```

Comment:
Does this really describe this package? Does it only deal with the Linked List structure?